Makefile

```diff
@@ -185,121 +199,393 @@ test-integration: test-integration-aws test-integration-clickhouse test-integrat
 test-integration: test-integration-gcp test-integration-influxdb test-integration-kafka test-integration-loki
 test-integration: test-integration-pulsar test-integration-splunk
 
+stop-test-integration: ## Stops all integration test infrastructure
+stop-test-integration: stop-integration-aws stop-integration-clickhouse stop-integration-elasticsearch
+stop-test-integration: stop-integration-gcp stop-integration-influxdb stop-integration-kafka stop-integration-loki
+stop-test-integration: stop-integration-pulsar stop-integration-splunk
+
+start-integration-aws:
+ifeq ($(CONTAINER_TOOL),podman)
+	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) create --replace --name test-integration-aws -p 8111:8111 -p 4568:4568 -p 4572:4572 -p 4582:4582 -p 4571:4571 -p 4573:4573 -p 6000:6000
+	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=test-integration-aws --name ec2_metadata \
+	 timberiodev/mock-ec2-metadata:latest
+	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=test-integration-aws --name localstack \
+	 -e SERVICES=kinesis:4568,s3:4572,cloudwatch:4582,elasticsearch:4571,firehose:4573 \
+	 localstack/localstack@sha256:f21f1fc770ee4bfd5012afdc902154c56b7fb18c14cf672de151b65569c8251e
+	$(CONTAINER_TOOL) run -d --$(CONTAINER_ENCLOSURE)=test-integration-aws --name mockwatchlogs \
+	 -e RUST_LOG=trace luciofranco/mockwatchlogs:latest
+else
+	$(CONTAINER_TOOL) $(CONTAINER_ENCLOSURE) create test-integration-aws
```

Comment:
These networks should have `vector_` namespaces so our contributors can better know where these artifacts are coming from.